docs/reference/project-structure.md

```diff
@@ -6,62 +6,66 @@ The AnyCloud CLI expects your project to be version controlled with `git`. Howev
+Deploying an AnyCloud `App` requires a [`Dockerfile`](https://docs.docker.com/engine/reference/builder/) file located within the top level folder of your project next to your `anycloud.json`. AnyCloud will expect the docker container described by the `Dockerfile` to have a HTTP server listening on port `8088` like the one [here](../tutorial.md#configure-your-project).
 
 ## anycloud.json
 
-AnyCloud deployments are configured using an `anycloud.json` file located within the top level folder of your project or repository, next to your [`Dockerfile`](). Each deployment will be keyed by a name and contain an array of deployment configurations that are described by the following fields: `region`, `vmType` and `credentials`. The string in the`credentials` field has to match one of the keys from your JSON  [credentials](credentials.md) file defined in `~/.anycloud/credentials.json`. The schema for `anycloud.json` is as follows:
+AnyCloud `Deploy Configs` are stored in an `anycloud.json` within the project directory, next to your `Dockerfile`, and configured using the CLI subcommands under `anycloud config`. Each `Deploy Config` consists of a name, or alias, to refer to it when creating or updating `Apps`. A `Deploy Config` consists of one or multiple of the following configurations: region, virtual machine type and an existing [`Credentials`](credentials.md) name or alias.
 
-```javascript
-{
-  "string": [
-    {
-      "region": "string",
-      "vmType": "string",
-      "credentials": "string"
-    }
-  ],
-  "string": [
-    {
-      "region": "string",
-      "vmType": "string",
-      "credentials": "string"
-    },
-    {
-      "region": "string",
-      "vmType": "string",
-      "credentials": "string"
-    }
-  ]
-  ...
-}
+```
+$ anycloud config add
+Name for new Deploy Config: staging
+Pick Credential to use:
+> mystartup-aws
+Region name: us-west-1
+Virtual Machine Type: t2.medium
+Do you want to add another region to this Deploy Config? n
+Successfully created "staging" Deploy Config
+
+$ anycloud config add
+Name for new Deploy Config: production
+Pick Credential to use:
+> mystartup-aws
+Region name: us-west-1
+Virtual Machine Type: t2.xlarge
+Do you want to add another region to this Deploy Config? n
+Successfully created "production" Deploy Config
+
+$ anycloud config list
+
+Deployment configurations:
+
+┌────────────┬─────────────────┬───────────┬───────────┐
+│ Name       │ Credential Name │ Region    │ VM Type   │
+├────────────┼─────────────────┼───────────┼───────────┤
+│ production │ azure-alantech  │ us-west-1 │ t2.xlarge │
+│ staging    │ azure-alantech  │ us-west-1 │ t2.medium │
```

Comment:
This doesn't look right. Should be `mystartup-aws` for the Credential Name instead of `azure-alantech`

Alternatively change the above to Azure stuff and then change the region and VM type to match.

Reply:
Ah good catch!